fvm/environment/event_emitter.go

```diff
@@ -295,8 +290,10 @@ func (collection *EventCollection) TotalEventCounter() uint32 {
 	return collection.eventCounter
 }
 
-// IsServiceEvent determines whether or not an emitted Cadence event is
-// considered a service event for the given chain.
+// IsServiceEvent determines whether an emitted Cadence event is considered a service event for the given chain.
+// An event is a service event if it is defined in the `systemcontracts` package allow-list.
+// All service events must be defined in system contracts.
```

Comment:
This is not use here neither is it enforced here, so there is no reason to put it In a comment here. 

If we want to enforce this, we should, but that is probably inside of `systemcontracts.ServiceEventsForChain`